pipeline.cpp

```diff
@@ -79,7 +82,39 @@ void DrawLine(Buffer2D<PIXEL> & target, Vertex* const triangle, Attributes* cons
  ************************************************************/
 void DrawTriangle(Buffer2D<PIXEL> & target, Vertex* const triangle, Attributes* const attrs, Attributes* const uniforms, FragmentShader* const frag)
 {
-    // Your code goes here
+    // get the bounding box of the triangle
+    int maxX = MAX3(triangle[0].x, triangle[1].x, triangle[2].x); // right-most value
```

Comment:
I didn't know that this was a thing

Reply:
Yeah me too! It sounds like it could be real helpful!